Emby.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -556,10 +556,11 @@ public string EncodeImage(string inputPath, DateTime dateModified, string output
                         _fileSystem.CreateDirectory(_fileSystem.GetDirectoryName(outputPath));
                         using (var outputStream = new SKFileWStream(outputPath))
                         {
-                            SKImageInfo imageInfo = new SKImageInfo(width,height);
-                            var pixmap = new SKPixmap(new SKImageInfo(width, height), resizedBitmap.GetPixels());
-                            pixmap.Encode(outputStream, skiaOutputFormat, quality);
-                            return outputPath;
+                            using (var pixmap = new SKPixmap(new SKImageInfo(width, height), resizedBitmap.GetPixels()))
```

Comment:
You don't need another indentation level for 2 using statements, you can do:
```cs
using (var stuff = foo)
using (var morestuff = foo)
{ // Use stuff and morestuff }

Reply:
Yes please, with Lukes indentation the lines are halfway my ultrawide.

Reply:
Will postpone the reformatting after #382 is merged, and then auto reformat.